tests/core/posix.c

```diff
@@ -164,8 +164,8 @@ void test_core_posix__p_regcomp_ignores_global_locale_ctype(void)
 		cl_fail("Expected locale to be switched to multibyte");
 	}
 
-	p_regcomp(&preg, "[\xc0-\xff][\x80-\xbf]", REG_EXTENDED);
-	regfree(&preg);
+	p_regcomp(&preg, "[\xc0-\xff][\x80-\xbf]", P_REG_EXTENDED);
```

Comment:
This really should be wrapped in `cl_must_pass`. Not the problem of this patchset, though

Reply:
Fixed.